sdk/identity/identity/CHANGELOG.md

```diff
@@ -52,72 +58,86 @@
 ## 4.0.0-beta.1 (2023-10-26)
 
 ### Features Added
+
 - Added `brokerOptions` in `InteractiveBrowserCredential` for authentication broker support such as WAM. This feature works along with the new `@azure/identity-broker` plugin package. Note that this feature is only available in node.
 - Added support for MSA passthrough in the `brokerOptions` of `InteractiveBrowserCredential`. Note this is only available for legacy 1st party applications.
 - Added `BrowserCustomizationOptions` for success and error messages in the `InteractiveBrowserCredential`.
 
 ### Breaking Changes
+
 - The `redirectUri` is no longer a required option for `InteractiveBrowserCredential` on Node.js. There's no API change, but this is a behavior change.
 
 ## 3.3.2 (2023-10-17)
 
 ### Bugs Fixed
-- Ensure `AzurePowershellCredential` calls PowerShell with the `-NoProfile`  and "-NonInteractive" flag to avoid loading user profiles for more consistent behavior. ([#27023](https://github.com/Azure/azure-sdk-for-js/pull/27023))
+
+- Ensure `AzurePowershellCredential` calls PowerShell with the `-NoProfile` and "-NonInteractive" flag to avoid loading user profiles for more consistent behavior. ([#27023](https://github.com/Azure/azure-sdk-for-js/pull/27023))
```

Comment:
I've been resisting making a large update to CHANGELOG but 🎉 I love that we're using the formatter and (maybe markdownlint?) for this now